exercises/say/src/example.h

```diff
@@ -1,6 +1,12 @@
 #ifndef SAY_H
 #define SAY_H
 #include <stdint.h>
+#include <stddef.h>
+
+typedef struct {
```

Comment:
I think this can go inside `example.c`, right? No need for it to be exposed to the caller if `map` is only used internally.

Reply:
Also, I think we've been following the convention of giving type names a suffix of `_t`, e.g. `map_t`.
This is common in non-POSIX environments to make it clear what is a type name and what is a variable name (because a lot of editors don't do syntax highlighting for custom types).

Reply:
Completely agree.

Reply:
FYI, the reason I mention non-POSIX environment is that names ending with `_t` are reserved in POISX. everywhere else it is a common idiom though.